pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [project]
 name = "nebula3-python"
-version = "3.5.0"
+version = "3.5.1"
```

Comment:
the next version will be 3.8.0

Reply:
Let's baby steps to release this first, then let's tag 3.8.0 💪